reposerver/src/main/scala/com/advancedtelematic/tuf/reposerver/db/Schema.scala

```diff
@@ -64,4 +65,18 @@ object Schema {
   }
 
   protected [db] val repoNamespaces = TableQuery[RepoNamespaceTable]
+
+  class PackageCommentTable(tag: Tag) extends Table[(RepoId, TargetFilename, TargetComment)](tag, "filename_comments") {
```

Comment:
Why not use a case class instead of a tuple, if using a case class to build the response anyways? `FilenameComment`

This should be called FilenameCommentsTable, to be consistent with the table name. Package is not a concept we use in tuf, there are targets.

Reply:
FilenameComment doesn't include the repo id.